pkgs/development/tools/pyenv/default.nix

```diff
@@ -6,13 +6,13 @@
 
 stdenv.mkDerivation rec {
   pname = "pyenv";
-  version = "2.3.32";
+  version = "2.3.34";
```

Comment:
We should probably merge:
- https://github.com/NixOS/nixpkgs/pull/272965
Instead of this PR.